docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -37,9 +37,71 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Authentication: Get OAuth Credentials\n",
+    "### Google Authentication Overview\n",
     "\n",
-    "The Google Semantic Retriever API lets you perform semantic search on your own data. Since it's **your data**, this needs stricter access controls than API Keys. Please follow [OAuth Quickstart](https://developers.generativeai.google/tutorials/oauth_quickstart) to setup OAuth authentication. Below are overview of steps from the documentation that are required.\n",
+    "The Google Semantic Retriever API lets you perform semantic search on your own data. Since it's **your data**, this needs stricter access controls than API Keys. Authenticate with OAuth through service accounts or through your user credentials.\n",
+    "\n",
+    "**Note**: At this time, the Google Generative AI Semantic Retriever API is [only available in certain regions](https://ai.google.dev/available_regions)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Authentication (Option 1): OAuth using service accounts\n",
```

Comment:
Should we also provide some reasonings to help the user to decide which option to choose?

Reply:
Added: This quickstart uses a simplified authentication approach for a testing environment, and service account setup are typically easier to start. Google Auth [service accounts](https://cloud.google.com/iam/docs/service-account-overview) let an application authenticate to make authorized Google API calls. For a production environment, learn about [authentication and authorization](https://developers.google.com/workspace/guides/auth-overview) before choosing the [access credentials](https://developers.google.com/workspace/guides/create-credentials#choose_the_access_credential_that_is_right_for_you) that are appropriate for your app.